internal/rbd/controllerserver.go

```diff
@@ -1324,10 +1311,18 @@ func (cs *ControllerServer) ControllerExpandVolume(
 		return nil, err
 	}
 
-	if rbdVol.isEncrypted() {
-		return nil, status.Errorf(codes.InvalidArgument, "encrypted volumes do not support resize (%s)",
-			rbdVol)
+	nodeExpansion := true
+	// Get the nodeexpansion flag set based on the volume mode
```

Comment:
can add more details to the comment on what we are doing here?

Reply:
sure :+1: